code/modules/admin/verbs/adminhelp.dm

```diff
@@ -341,336 +145,121 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
-	if(!admin_datum)
-		message_admins("[usr] sent a request to interact with the ticket window without sufficient rights.")
-		log_admin_private("[usr] sent a request to interact with the ticket window without sufficient rights.")
-		return
-	if(!check_rights(R_ADMIN))
-		message_admins("[usr] sent a request to interact with the ticket window without sufficient rights. (Requires: R_ADMIN)")
-		log_admin_private("[usr] sent a request to interact with the ticket window without sufficient rights.")
-		return
-	//Doing action on a ticket claims it
-	var/claim_ticket = CLAIM_DONTCLAIM
-	switch(action)
-		if("sendpm")
-			usr.client.cmd_ahelp_reply_instant(initiator, params["text"])
-			claim_ticket = CLAIM_CLAIMIFNONE
-		if("reject")
-			Reject()
-			claim_ticket = CLAIM_OVERRIDE
-		if("mentorhelp")
-			MHelpThis()
-			claim_ticket = CLAIM_OVERRIDE
-		if("close")
-			Close()
-			claim_ticket = CLAIM_OVERRIDE
-		if("resolve")
-			Resolve()
-			claim_ticket = CLAIM_OVERRIDE
-		if("markic")
-			ICIssue()
-			claim_ticket = CLAIM_OVERRIDE
-		if("retitle")
-			Retitle()
-		if("reopen")
-			Reopen()
-			claim_ticket = CLAIM_OVERRIDE
-		if("moreinfo")
-			admin_datum.admin_more_info(get_mob_by_ckey(initiator.ckey))
-		if("playerpanel")
-			admin_datum.show_player_panel(get_mob_by_ckey(initiator.ckey))
-		if("viewvars")
-			usr.client.debug_variables(get_mob_by_ckey(initiator.ckey))
-		if("subtlemsg")
-			usr.client.cmd_admin_subtle_message(get_mob_by_ckey(initiator.ckey))
-		if("flw")
-			admin_datum.admin_follow(get_mob_by_ckey(initiator.ckey))
-		if("traitorpanel")
-			admin_datum.show_traitor_panel(get_mob_by_ckey(initiator.ckey))
-		if("viewlogs")
-			show_individual_logging_panel(get_mob_by_ckey(initiator.ckey))
-		if("smite")
-			usr.client.smite(get_mob_by_ckey(initiator.ckey))
-	if(claim_ticket == CLAIM_OVERRIDE || (claim_ticket == CLAIM_CLAIMIFNONE && !claimed_admin))
-		Claim()
-
-/datum/admin_help/proc/MessageNoRecipient(msg)
+/datum/help_ticket/admin/key_name_ticket(mob/user)
+	return key_name_admin(user)
+
+/datum/help_ticket/admin/message_ticket_managers(msg)
+	message_admins(msg)
+
+/datum/help_ticket/admin/MessageNoRecipient(msg, add_to_ticket = TRUE)
 	var/ref_src = "[REF(src)]"
 
 	//Message to be sent to all admins
 	var/admin_msg = "<span class='adminnotice'><span class='adminhelp'>Ticket [TicketHref("#[id]", ref_src)]</span><b>: [LinkedReplyName(ref_src)] [FullMonty(ref_src)]:</b> <span class='linkify'>[keywords_lookup(msg)]</span></span>"
 
-	AddInteraction("red", msg, initiator_key_name, claimed_admin_key_name, "You", "Administrator")
+	if(add_to_ticket)
+		AddInteraction("red", msg, initiator_key_name, claimee_key_name, "You", "Administrator")
 	log_admin_private("Ticket #[id]: [key_name(initiator)]: [msg]")
 
 	//send this msg to all admins
 	for(var/client/X in GLOB.admins)
 		if(X.prefs.toggles & SOUND_ADMINHELP)
-			SEND_SOUND(X, sound('sound/effects/adminhelp.ogg'))
+			SEND_SOUND(X, sound(reply_sound))
```

Comment:
oh god don't give admins a way of changing this

Reply:
not what this is, it's just a proc argument

Reply:
❓

Reply:
`reply_sound` is a var on /datum/help_ticket
/datum/help_ticket/mentor changes the var to the MENTOR sound (bikehorn.ogg)
reply_sound should NOT be varedited, and if an admin does that, that's their problem.